doc/usage.rst

```diff
@@ -20,3 +24,19 @@ When executing partup, the following options can be specified:
 -p, --prefix=PREFIX     Path to prefix all file URIs with in the layout configuration
 -s, --skip-checksums    Skip checksum verification for all input files
 -v, --version           Print the program version and exit
+
+Supported Output Devices
+------------------------
+
+Note, that writing is currently only supported for MMC devices. The device must
```

Comment:
Maybe mention eMMC and the boot partitions here.

Reply:
Done.

Added a more extensive list of supported devices.